docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+  dockerpractice:
+    build: ./ops-tools/docker/images/app/
+    command: java -jar ./target/docker-practice-java-0.0.1-SNAPSHOT.jar
+    environment:
+      POSTGRES_USER: postgres
```

Comment:
These environments for the Backend part, right? Does your application use those environments bypassing `application.properties`, right?